de/graphNorms.py

```diff
@@ -0,0 +1,98 @@
+from .factorization import cellLineFactorization, MatrixSubtraction
+import matplotlib.pyplot as plt
+import numpy as np
+from math import ceil
+
+
+def calcNorms(cellLine1, cellLine2):
+    
+    _, difference_norm, w1, w2 = MatrixSubtraction(cellLine1, cellLine2)
```

Comment:
Why is the difference norm calculated inside this function, but then you have another separate function to give the single cell line norms?